### PR TITLE
LPS-165401 LayoutClassedModelUsage's containerType field not set correctly on impor

### PR DESCRIPTION
**Pull request by vendeltoreki**
Forwarding from: https://github.com/liferay-staging/liferay-portal/pull/136

* Note about commit https://github.com/liferay-echo/liferay-portal/commit/fb025ce3398894ea2922946f9fede3818f3c76cb : `classNameIds` can be different between instances, so conversion during import might still be necessary.

cc: @mbowerman

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by mbowerman** [comment]
> Note about commit https://github.com/liferay-echo/liferay-portal/commit/fb025ce3398894ea2922946f9fede3818f3c76cb : classNameIds can be different between instances, so conversion during import might still be necessary.

I do not agree with this. I know that classNameIds can be different between instances (in fact, that's the entire point of https://github.com/liferay-echo/liferay-portal/commit/711d6939e1d2c94338b71071fed3a3ea501476f7), but the convert-and-reconvert logic of this classNameId field is not necessary.

Conversion during import is not necessary because it is handled automatically in PortletDataContext. Furthermore, even if it _was_ necessary, the previous logic that we had there wouldn't have worked anyway. `_portal.getClassNameId(layoutClassedModelUsage.getClassName())` will _always_ return the exact same result as `layoutClassedModelUsage.getClassNameId()`.

Let me defend each of these statements in turn.

> Conversion during import is not necessary because it is handled automatically in PortletDataContext.
The PortletDataContextImpl class has logic to automatically handle the conversion of a field named `classNameId` when exporting and importing between instances. It assigns an `attached-class-name` attribute during export with the className. Then it reads that `attached-class-name` during import and sets the `classNameId` field of the object to the corresponding classNameId of that className on the import side.

You can see where this happens on the export side [here](https://github.com/liferay/liferay-portal/blob/1dcf56d8a945ed677d6db6110a7f54c822adc852/modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java#L2745-L2754) and where it happens on the import side [here](https://github.com/liferay/liferay-portal/blob/1dcf56d8a945ed677d6db6110a7f54c822adc852/modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java#L1231-L1241).

> Furthermore, even if it _was_ necessary, the previous logic that we had there wouldn't have worked anyway. `_portal.getClassNameId(layoutClassedModelUsage.getClassName())` will _always_ return the exact same result as `layoutClassedModelUsage.getClassNameId()`.
`layoutClassedModelUsage.getClassName()` is equivalent to calling `layoutClassedModelUsage.getClassNameId()` and then converting it to a className by calling `PortalUtil.getClassName()`. You can see the logic of `layoutClassedModelUsage.getClassName()` [here](https://github.com/liferay/liferay-portal/blob/1dcf56d8a945ed677d6db6110a7f54c822adc852/modules/apps/layout/layout-service/src/main/java/com/liferay/layout/model/impl/LayoutClassedModelUsageModelImpl.java#L545-L552).

In other words, `_portal.getClassNameId(layoutClassedModelUsage.getClassName())` does the following:
1. Starts with `layoutClassedModelUsage.getClassNameId()`.
2. Converts that classNameId to a className by querying the `ClassName_` table in the database.
3. Converts that className back to a classNameId by querying the `ClassName_` table in the database.

Since the `ClassName_` table's values and classNameIds are both guaranteed to be unique, steps 2-3 will always be a noop collectively. Therefore we do not need to perform them, and can simply stick with step 1.

**Comment by ealonso** [comment]
Just started reviewing :)

**Comment by ealonso** [comment]
ci:forward

**Comment by liulukiwi** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward
